rfcs/ClientControlledNullability.md

```diff
@@ -134,6 +134,17 @@ is non-nullable, so it will feel familiar to GraphQL developers.
 
 ## Use cases
 
+#### When handling partial results
+
+It is often unclear how to handle partial results.
+
+- What elements are essential to having an adequate developer experience? How many elements can fail before you 
+  should just show an error message instead?
```

Comment:
this feels like more of a problem statement than a use-case, in particular providing more details on this point
> While the schema can have nullable fields for valid reasons (such as federation), in some cases the client wants to decide if it accepts a `null` value for the result **to simplify the client-side logic**.

it's also got some low key overlap with https://github.com/twof/graphql-spec/pull/9/

I think the focus of the proposal should be on the solution and implementation/behavior. should we start an appendix that describes some of the issues mentioned in the problem statement in more detail, to keep the initial problem statement short and clear? I'm not sure that we need a subsection for each of these points, but I'm curious what other folks think.